src/renderer/components/execute/Results.tsx

```diff
@@ -30,36 +144,49 @@ const Results = () => {
         <div>
           <h1>Execution Results</h1>
         </div>
-        <Table className="table">
-          <thead>
-            <tr key={results.id}>
-              <th>Name</th>
-              <th>ID</th>
-              <th>Database</th>
-              <th>Type</th>
-              <th>Result</th>
-              <th> Message </th>
-            </tr>
-          </thead>
-          <tbody>
-            {results.rules.map((value) => {
-              return (
-                <>
-                  {passFail(value) ? (
-                    <AllPass Rule={value} />
+        {execution.rules.map((rule) => {
+          return (
+            <Accordion className="accordionClass">
+              <AccordionItem eventKey="0">
+                <AccordionHeader id="header">
+                  {passFail(rule) ? (
+                    <>
+                      {rule.name} <CheckLg className="Check" />
+                    </>
                   ) : (
-                    <Displaytests Rule={value} />
+                    rule.name
                   )}
-                </>
-              );
-            })}
-          </tbody>
-        </Table>
+                </AccordionHeader>
+                <AccordionBody>
+                  {rule.unitTests.map((Test) => {
+                    return (
+                      <Table className="Results-Table">
```

Comment:
Instead of creating the table for each mapped unit test, what if we created the Table outside of the map and then just added a <tr> for each element? This might also be related to how the intermediate rows of the table have a thicker border because there are two tables are stacked on each other